src/VisualStudio/Core/Test/Snippets/SnippetTestState.vb

```diff
@@ -37,7 +37,9 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Snippets
             Workspace.TryApplyChanges(Workspace.CurrentSolution.WithOptions(Workspace.Options _
                     .WithChangedOption(InternalFeatureOnOffOptions.Snippets, True)))
 
-            Dim mockSVsServiceProvider = New Mock(Of SVsServiceProvider)
+            Dim mockSVsServiceProvider = New Mock(Of SVsServiceProvider)(MockBehavior.Strict)
+            mockSVsServiceProvider.Setup(Function(s) s.GetService(GetType(SVsTextManager))).Returns(Nothing)
```

Comment:
Wish we also had the loose mock here as well. A random implementation change elsewhere doesn't really feel like it should trigger a requirement to add an explicit "Returns(Nothing)" if the test would have already passed.

Reply:
> A random implementation change elsewhere doesn't really feel like it should trigger a requirement to add an explicit "Returns(Nothing)" if the test would have already passed.

This is why we don't use Moq.